app/components/AiChatBase.tsx

```diff
@@ -0,0 +1,156 @@
+              <Dialog.Description hidden={true}>
+                Presagio chatbot window
+              </Dialog.Description>
+              <div className="border-b border-outline-low-em p-4">
+                <div className="flex items-center justify-between">
+                  <div className="flex items-center space-x-2">
+                    <Image alt="ai stars" width={14} height={14} src={aiStarsSvg} />
+                    <Dialog.DialogTitle className="font-semibold">
+                      Ask Wizard
+                    </Dialog.DialogTitle>
+                    <Tag colorScheme="info">Experimental</Tag>
+                  </div>
+                  <Dialog.Close asChild>
+                    <IconButton
+                      size="sm"
+                      className="rotate-90"
+                      variant="ghost"
+                      name="arrow-double-right"
+                    />
+                  </Dialog.Close>
+                </div>
+              </div>
+              <ScrollArea ref={setScrollAreaElement} className="h-[460px] md:h-[536px]">
+                <div className="space-y-1.5 px-4 pb-32 pt-4">{children}</div>
+              </ScrollArea>
+              <div className="absolute bottom-0 w-full rounded-b-16 border-t border-surface-surface-1 bg-surface-surface-0 px-4 pb-4 pt-2">
+                <div className="flex items-center justify-center">
+                  <a
+                    onClick={() => {
+                      trackEvent(FA_EVENTS.MARKET.AI_CHAT.GET_BETA_ACCESS);
+                    }}
+                    href={MARKETING_LINK}
+                    className="mb-1 hover:underline"
+                    target="_blank"
+                  >
+                    Get beta access to Presagio AI ✨
+                  </a>
+                  <Icon name="arrow-top-right" size={12} />
+                </div>
+                <Input
+                  placeholder="Ask anything. Available soon."
+                  className="w-full"
+                  disabled
+                />
+                <p className="w-full text-center text-xs text-text-low-em">
+                  AI-generated, for reference only.
+                </p>
+              </div>
+            </Dialog.Content>
+          </Dialog.Portal>
+
+          <Dialog.Trigger asChild>
+            <button
+              onClick={() => {
+                trackOnClickEvents.map(event => trackEvent(event));
+              }}
+              className={twMerge(
+                'flex size-12 items-center justify-center rounded-100 bg-transparent shadow-1 outline-outline-primary-low-em backdrop-blur-sm transition-colors duration-700 hover:bg-outline-primary-base-em focus:bg-outline-primary-base-em md:size-16',
+                'data-[state=open]:bg-outline-primary-base-em data-[state=open]:shadow-2'
+              )}
```

Comment:
Check the comments on this file for further context on the main function

```suggestion
 className={twMerge(
                'flex size-12 items-center justify-center rounded-100 shadow-1 outline-outline-primary-low-em backdrop-blur-sm transition-colors duration-700 hover:bg-outline-primary-base-em focus:bg-outline-primary-base-em md:size-16',
                'data-[state=open]:bg-outline-primary-base-em data-[state=open]:shadow-2',
                borderAndBgColor
              )}
```